drivers/scheduler/k8s/k8s.go

```diff
@@ -24,7 +24,11 @@ import (
 )
 
 // SchedName is the name of the kubernetes scheduler driver implementation
-const SchedName = "k8s"
+const (
+	SchedName = "k8s"
```

Comment:
nit: SchedName should be a separate constant.

Define a different enum for core and storage